VultisigApp/VultisigApp/Chains/UTXOChainsHelper.swift

```diff
@@ -69,8 +69,13 @@ class UTXOChainsHelper {
         else {
             return .failure(HelperError.runtimeError("public key \(derivedPubKey) is invalid"))
         }
+        let address = coin.deriveAddressFromPublicKey(publicKey: publicKey)
```

Comment:
do we need to add prefix again when we setting address into WalletCore tx input?

Reply:
No, it works with prefixes and without the prefix for sending and receiving.